frontend/app/services/lookup-service.server.ts

```diff
@@ -13,16 +13,16 @@ const preferredLanguageSchema = z.object({
 });
 
 const countrySchema = z.object({
-  code: z.string(),
-  nameEn: z.string(),
-  nameFr: z.string(),
+  countryId: z.string(),
+  nameEnglish: z.string(),
+  nameFrench: z.string(),
 });
 
 const regionSchema = z.object({
-  code: z.string(),
-  country: countrySchema,
-  nameEn: z.string(),
-  nameFr: z.string(),
+  provinceTerritoryStateId: z.string(),
+  countryId: z.string(),
+  nameEnglish: z.string(),
+  nameFrench: z.string(),
 });
```

Comment:
Wasn't sure where to put this comment but consider defining a new schema/object that will be used to specifically consume the API. Then we can map this object to our own application domain model in the service before returning it.

That way, if the API were to ever change it's fields (ie. if we use Interop and they return their responses with NIEM naming conventions), we won't have to change the name of the fields throughout the application.

Reply:
Added mapped return